JitsiConferenceEventManager.js

```diff
@@ -300,6 +300,23 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function() {
                 JitsiConferenceEvents.USER_STATUS_CHANGED, id, status);
         });
 
+    chatRoom.addListener(XMPPEvents.JSON_MESSAGE_RECEIVED,
+        (from, payload) => {
+            const participant = conference.getParticipantById(
+                from.split('/')[1]);
+
+            if (participant) {
+                conference.eventEmitter.emit(
+                    JitsiConferenceEvents.ENDPOINT_MESSAGE_RECEIVED,
+                    participant, payload);
+            } else {
+                logger.warn(
+                    'Ignored ENDPOINT_MESSAGE_RECEIVED for not existing '
+                    + `participant: ${from}`,
```

Comment:
Technically we ignored an XMPPEvents.JSON_MESSAGE_RECEIVED. I think it might be useful to change the message so that logs from the two places can be distinguished.